ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-sno-ranGen.yaml

```diff
@@ -184,3 +184,18 @@ spec:
 #                user:
 #                  name: root
 #     ---- sources needed for updating CRI-O workload-partitioning ends here ----
+
+    # ---- Sources needed to configure IPsec encryption for external traffic ----
+    # 1. Provide the following certificate files:
+    #    * left_server.p12
+    #    * ca.pem
+    # 2. Set the following values in the `ipsec-endpoint-config.yml`:
+    #    * `$clusterNode` - the node IP address or DNS hostname
+    #    * `$externalHost` - the external host IP address or DNS hostname
+    #    * `$externalAddress` - the IP address or subnet of the external host
+    # 3. Run the `build.sh` script to create the machine config
+    # - fileName: enable-ipsec.yaml
+    #   policyName: "config-policy"
+    # - fileName: 99-ipsec-master-endpoint-config.yaml
```

Comment:
This file does not exist unless MC yaml is added directly to ztp/source-cr dir.

Reply:
Since the script generates a MC you can choose where to store it, please note this feature requires a pre step and also an optional feature that's why it is stored in optional dir.
Please follow the README and the official documentation https://docs.openshift.com/container-platform/4.15/networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.html#nw-ovn-ipsec-external_configuring-ipsec-ovn